AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -91,6 +91,7 @@ techadvisor.co.uk,forum.notebookreview.com,computerworld.com,skysports.com#$#bod
 ||cdn.privacy-mgmt.com^$important,domain=promipool.de|android-hilfe.de|wsj.com
 !
 !
+superhaber.tv##.privacy-banner:not(body):not(html)
```

Comment:
please add the domain to existing rule `##.privacy-banner`

Reply:
Okay.